src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,61 @@
 import React from 'react';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+import { Post } from '../../types/Post';
+
+import './PostInfo.scss';
+
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+import commentsFromServer from '../../api/comments';
+import usersFromServer from '../../api/users';
+
+type Props = {
+  post: Post,
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const {
+    userId,
+    id,
+    title,
+    body,
+  } = post;
+
+  const neededComments = commentsFromServer.filter(comment => (
+    comment.postId === id
+  ));
+
+  const neededUser = usersFromServer.find(user => (
+    user.id === userId
+  ));
+
```

Comment:
my friend)  cheat it))) user test is a luck. comments cant pass.  cause they pass in test params whole object with user and coments and trying to get them back.